.github/stale.yml

```diff
@@ -1,16 +1 @@
-# Number of days of inactivity before an issue becomes stale
-daysUntilStale: 1
-# Number of days of inactivity before a stale issue is closed
-daysUntilClose: 2
-# Issues with these labels will never be considered stale
-exemptLabels:
-  - 'Payment Status: Paid'
-  - pinned
-  - security
-# Comment to post when marking an issue as stale. Set to `false` to disable
-markComment: >
-  This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
-  for your contributions.
-# Comment to post when closing a stale issue. Set to `false` to disable
-closeComment: false
+#move patch to P0X commit
```

Comment:
# - [ ] .github/stale.yml[]()@CeylonJohn P0X []()make check -- Execute validation checks; also used in continuous integration.
make fix -- Perform automatic fixes where possible
make update-auto -- Run automatic updates from external sources, executed regularly (GitHub action)
make add-token https://etherscan.io/token/0xd8ce941666e429bbd969c1af11f474b505c69bc5#readContract#F13 -- Create info.json file as asset template.
make add-tokenlist https://etherscan.io/token/0xd8ce941666e429bbd969c1af11f474b505c69bc5#readContract#F13 -- Adds a token to tokenlist.json.
make add-tokenlist-extended https://etherscan.io/token/0xd8ce941666e429bbd969c1af11f474b505c69bc5#readContract#F13 --